src/MainWindow/main_window.cpp

```diff
@@ -1220,6 +1271,24 @@ bool MainWindow::confirmExitProgram() {
               QMessageBox::No | QMessageBox::Yes) == QMessageBox::Yes);
 }
 
+void MainWindow::setVisibleRefreshButtons(bool visible) {
```

Comment:
Should probably rename to indicate this only works with pin planner docks. Anything that uses PrepareTab can add this refresh button so currently the function name seems to imply it will update all refresh buttons.

Reply:
PrepareTab updated. Now it doesn't related to pin planner.